src/solver/type_getters.jl

```diff
@@ -143,7 +142,7 @@ function get_spaces(parsed_args, params, comms_ctx)
             lat = Array(data["latitude"])
             # Apply Smoothing
             smooth_degree = Int(parsed_args["smoothing_order"])
-            esmth = imfilter(zlevels, Kernel.gaussian(smooth_degree))
+            esmth = gaussian_smooth(smooth_degree)
```

Comment:
Are we missing the array as an `arg` here?

Reply:
Yes we are